src/openzaak/components/autorisaties/admin_views.py

```diff
@@ -128,13 +148,18 @@ def get_initial_for_component(
                         "informatieobjecttypen": relevant_ids,
                     }
                 )
-            initial.append(_initial)
+        initial.append(_initial)
 
     elif component == ComponentTypes.brc:
         besluittype_ids = set(BesluitType.objects.values_list("id", flat=True))
         relevant_ids = set(related_objs.values())
 
-        _initial = {}
+        _initial = {"externe_typen": _related_objs_external}
```

Comment:
The same remarks as above apply

Reply:
Besluittypen do not have `vertrouwelijkheidaanduiding`, so this one should be fine right?

Reply:
Yeah, my copy-paste mania got a bit too much :-)